pages/price-feeds/best-practices.mdx

```diff
@@ -51,7 +54,7 @@ The threat model for integrating protocols should assume that adversaries see pr
 In this threat model, protocol designers should avoid situations where a Pyth price update must race against an adversary's transaction.
 Adversaries are highly likely to win these races, as they have a head start, and sophisticated adversaries can additionally optimize their network latencies or pay miners for priority blockspace.
 
-This situation is analogous to market making in traditional finance.
+This situation is analogous to market-making in traditional finance.
```

Comment:
I think maybe we should cut this whole section about the market making analogy and instead simply offer the specific latency mitigations (?)

Reply:
Yes, I agree with it.